containers-template/src/index.ts

```diff
@@ -0,0 +1,62 @@
+import { Container, loadBalance, getContainer } from "@cloudflare/containers";
+
+export class MyContainer extends Container {
+  defaultPort = 8080;
+  sleepAfter = "2m";
+  envVars = {
+    MESSAGE: "I was passed in via the container class!",
```

Comment:
super nit - the way this is interpolated the "!" results in a grammar error

Maybe wrap this in internal quotes?

Reply:
Sounds good, this is how it prints now:
`Hi, I'm a container and this is my message: "I was passed in via the container class!", my instance ID is: d037394b-e266-4e55-8b7c-dbd2cb99354d`